woocommerce/woocommerce-android
# Issue/2189 update product tags

Fixes #2189 by adding logic to display and update product tags.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/86094076-a525c780-bacd-11ea-86d7-108bec5c08ac.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/86094081-a6ef8b00-bacd-11ea-9b5f-399cf8dde308.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/86094142-bf5fa580-bacd-11ea-9128-744636dad504.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/86094149-c25a9600-bacd-11ea-8c8a-021a23167dfe.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/86094231-ddc5a100-bacd-11ea-8cdb-1751a6d2e1ca.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/86094240-e027fb00-bacd-11ea-975a-64d1eb0176d4.png" width="300"/>

#### To test
- Click on the tags section of a product and verify that the tags of the site are displayed correctly.
- Verify that pagination works correctly.
- Verify that sorting matches the UI in the web. (i.e. by name asc).
- Verify that you are able to select multiple tags from the list.
  - Notice that the `DONE` menu button is only visible when a new tag is selected.
  - Click on the back menu button and notice that a discard dialog is displayed.
  - Click on the `Discard` button and notice that the changes you made are discarded in the product detail screen.
- Click on the tags section again.
  - Select a new tag from the list and notice that the selected tag is displayed in the Chips UI.
  - Click on the `DONE` menu button and notice that the tag you selected is displayed in the product detail screen.
  - Click on `UPDATE` menu button and verify that the tag is successfully updated.
- Click on the tag section of a product.
  - De-select all the selected tag and verify that the Chips UI is no longer displayed. 
  - Click on the `DONE` menu button.
  - Click on the `UPDATE` menu button and verify that the tag is successfully updated.

Update release notes:
